README.MD

```diff
@@ -264,31 +264,42 @@ const signature = workerAccount.signEIP712TypedData(stakeRequestTypedData);
 ### Facilitator calls acceptStakeRequest
 
 ```js
+const { AbiBinProvider } = BrandedToken;
 const Mosaic = require('@openstfoundation/mosaic.js');
 const hashLockInstance = Mosaic.Helpers.StakeHelper.createSecretHashLock();
 const gatewayContractInstance = Mosaic.Contracts.getEIP20Gateway(web3Provider, gatewayAddress);
-let bountyAmountInWei;
-gatewayContractInstance.methods.bounty().call().then(function(result){
-  bountyAmountInWei = result;
-});
-const Facilitator = BrandedToken.Helpers.Facilitator;
-const facilitatorInstance = new Facilitator(web3Provider, eip20Token, brandedTokenAddress, gatewayComposerAddress, facilitator);
-txOptions = {
-  from: facilitator,
-  gas: '7500000'
-};    
-let stakeRequestHash; // stakeRequestHash unique per stake request stored in GatewayComposer.stakeRequests mapping
-facilitatorInstance.acceptStakeRequest(
-  stakeRequestHash,
-  signature,
-  bountyAmountInWei,
-  eip20TokenAbi,
-  hashLockInstance.hashLock,
-  web3Provider,
-  txOptions
-).then(function(){
-  console.log("Successfully completed acceptStakeRequest!");
-})
+
+const stake = async () => {
+  const bountyAmountInWei = await gatewayContractInstance.methods.bounty().call();
```

Comment:
I suggest below PR to merge before this PR. Because we have updated with Gateway contract interact in the readme.
```
gatewayContractInstance.getBounty().then(function(result){
  bountyAmountInWei = result;	
});
```
https://github.com/OpenSTFoundation/brandedtoken.js/pull/79/files

Reply:
OK. The other PR could fix the racing condition, then I don't need to make any changes to the README here.

Reply:
Along with racing fix this PR also changes facilitatorInstance.acceptStakeRequest() signature.
I suggest you please keep your changes.
We can merge below PR#79 first and update gatewayContractInstance.methods.bounty() to gatewayContractInstance.getBounty().
https://github.com/OpenSTFoundation/brandedtoken.js/pull/79/files
Makes sense to you?